src/commands/dev/index.js

```diff
@@ -28,24 +28,27 @@ const SERVER_POLL_INTERVAL = 1e3
 // 20 seconds
 const SERVER_POLL_TIMEOUT = 2e4
 
-const startFrameworkServer = async function ({ settings, log, exit }) {
-  if (settings.noCmd) {
-    const server = new StaticServer({
-      rootPath: settings.dist,
-      name: 'netlify-dev',
-      port: settings.frameworkPort,
-      templates: {
-        notFound: path.join(settings.dist, '404.html'),
-      },
-    })
+const startStaticServer = async ({ settings, log }) => {
+  const server = new StaticServer({
+    rootPath: settings.dist,
+    name: 'netlify-dev',
+    port: settings.frameworkPort,
+    templates: {
+      notFound: path.join(settings.dist, '404.html'),
+    },
+  })
 
-    await new Promise((resolve) => {
-      server.start(function onListening() {
-        log(`\n${NETLIFYDEVLOG} Server listening to`, settings.frameworkPort)
-        resolve()
-      })
+  await new Promise((resolve) => {
+    server.start(function onListening() {
```

Comment:
Looking at https://github.com/nbluis/static-server/blob/38994a482a382019a99690cf7a5dfdbe5c332c61/server.js#L114, it seems like we might be able to do:

```js
await promisify(server.start.bind(server))()
log(`\n${NETLIFYDEVLOG} Server listening to`, settings.frameworkPort)
```

What do you think?

Reply:
That work 🎉  https://github.com/netlify/cli/pull/2824/commits/70ad3a89062817fe9a961851870bcfb22049b7df